src/test/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfigTest.java

```diff
@@ -86,4 +88,30 @@ public void testValidator() {
         assertThrows(ConfigException.class, () -> validator.ensureValid(PrometheusMetricsReporterConfig.LISTENER_CONFIG, "randomhttp://:8080random"));
         assertThrows(ConfigException.class, () -> validator.ensureValid(PrometheusMetricsReporterConfig.LISTENER_CONFIG, "randomhttp://:8080"));
     }
+
+    @Test
+    public void testIsListenerEnabled() {
+        Map<String, Boolean> props = new HashMap<>();
+        props.put(PrometheusMetricsReporterConfig.LISTENER_ENABLE_CONFIG, true);
+        PrometheusMetricsReporterConfig config = new PrometheusMetricsReporterConfig(props);
+        Optional<HTTPServer> httpServerOptional = config.startHttpServer();
+
+        assertTrue(httpServerOptional.isPresent());
+        assertTrue(config.isListenerEnabled());
+
+        HTTPServer httpServer = httpServerOptional.get();
+        httpServer.close();
```

Comment:
This can be simplified in `httpServerOptional.ifPresent(HTTPServer::close);`

Reply:
Simplified. I removed the empty line between the assertions and closing the server. Is this ok?